<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1048</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="select_Widget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>751</width>
     <height>661</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="select_lineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="searchprev">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>890</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;이전</string>
   </property>
   <property name="icon">
    <iconset theme="화살표">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSearch">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>이름</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>아파트동호수</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>아이디</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>연락처</string>
    </property>
   </item>
  </widget>
  <widget class="QCommandLinkButton" name="searchnext">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>890</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>다음&gt;</string>
   </property>
   <property name="icon">
    <iconset theme="화살표">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labePlus">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>880</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertbtn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>추가</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDelAll">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>970</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>전부삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUpdate">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>970</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>최신등록순</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
